server/lib/Room.js

```diff
@@ -63,6 +63,88 @@ const ROUTER_SCALE_SIZE = config.routerScaleSize || 40;
 
 class Room extends EventEmitter
 {
+
+	static getLeastLoadedRouter(mediasoupWorkers, peers, mediasoupRouters)
+	{
+
+		const routerLoads = new Map();
+
+		const workerLoads = new Map();
+
+		for (const peer of peers.values())
+		{
+			const routerId = peer._routerId;
+
+			if (routerLoads.get(routerId))
```

Comment:
The Map has a method `has` for this purpose. This check would skip if the key exists, but the value is falsey, change to `routerLoads.has(routerId)`.